### DISCUSSION TITLE
RealBasic video super resolution 足球场地画面

### DISCUSSION THREAD

**Discussion by zhangjiarui130** [open-mmlab/mmediting#1277] [Ideas]
![image](https://user-images.githubusercontent.com/19943362/171154757-a7a15d95-9e29-4783-9ef2-1c891f04e349.png)

We have trained the RealBasicVSR 2X model by ourselves using the official method. After the over-splitting, the scene has a lot of noise and the picture effect is not improved. Can we modify some parameters of the script for adjustment
我们使用官方的方法，自己训练了RealBasicVSR 2X的模型，超分后场地画面画面噪点较多，画面效果没有提升，是否可以修改脚本的一些参数进行调整

**Comment by mm-assistant** [comment]
We recommend using English or English & Chinese for issues so that we could have broader discussion.

**Comment by wangruohui** [comment]
Hi,

Can you please upload the training logs for our reference?

**Comment by zhangjiarui130** [comment]
[train_log.tar.gz](https://github.com/open-mmlab/mmediting/files/8821212/train_log.tar.gz)

**Comment by zhangjiarui130** [comment]
https://github.com/open-mmlab/mmediting/issues/908#issuecomment-1144479723
this is training logs ，Please provide modification suggestions

**Comment by ckkelvinchan** [comment]
Hello I haven't tried x2 models before. How many frames did you use during inference?

**Comment by zhangjiarui130** [comment]
1. By modifying the scale values of RealBasicVSR_WOGAN_C64B20_2X30X8_LR1E-4_300K_REDS and RealBasicVSR_C64B20_1X30X8_LR5E-5_150K_REDS, Retraining using REDS 'Train_Sharp dataset has approximately 1110 images
2. May I ask whether the above method is feasible or whether the model effect can be optimized by adjusting parameters
3. for example modify RealBasicVSR_WOGAN_C64B20_2X30X8_LR1E-4_300K_REDS and diff 
![image](https://user-images.githubusercontent.com/19943362/173177722-846a55cb-50d4-4e43-a85f-ff9f9d730433.png)

**Comment by zengyh1900** [comment]
@Z-Fran 
Please check this issue.
